rules/possible_errors.js

```diff
@@ -1 +1,18 @@
-module.exports = {};
+module.exports = {
+  'no-console': 'off', // https://github.com/dwyl/goodparts/issues/16
```

Comment:
_not_ a fan of having `console.log`s left in the code, because allowing one rapidly turns into _many_ which creates (_a lot of_) _noise_ in the Logs ...
https://github.com/dwyl/goodparts/issues/16#issuecomment-248952858
Would _much_ prefer we implement a Logging solution up-front.